neuralmonkey/runners/runner.py

```diff
@@ -62,7 +66,9 @@ def __init__(self, all_coders, fetches, vocabulary, postprocess) -> None:
 
     def next_to_execute(self) -> NextExecute:
         """Get the feedables and tensors to run."""
-        return self.all_coders, self._fetches, {}
+        return (self.all_coders,
+                self._fetches,
+                [{} for _ in range(self._num_sessions)])
```

Comment:
Nedalo by se tadyto generování tohohle `empty_feed_dicts` přesunout do obecnýho `Executable` a tady to volat? Ušetří se tim docela dost duplikovanýho kódu.

Reply:
Dalo by se vracet `None` a postaral by se o to `tf_manager`.

Reply:
DONE